engineering-toolkit/eslint-config/src/typescript.js

```diff
@@ -78,9 +78,11 @@ export function typescript(config, ...overrides) {
         ],
         "@typescript-eslint/no-use-before-define": "warn",
         "@typescript-eslint/prefer-function-type": "warn",
+        "@typescript-eslint/no-empty-object-type": "warn",
         "default-param-last": "off",
         "no-magic-numbers": "off",
         "no-use-before-define": "off",
+        "no-unused-vars": "off",
```

Comment:
Disabling this rule is recommended when using `@typescript-eslint/no-unused-vars` rule.
https://typescript-eslint.io/rules/no-unused-vars/#how-to-use